src/dispatcher/mod.rs

```diff
@@ -150,7 +157,8 @@ impl Dispatcher {
         Ok(())
     }
 
-    pub fn up(&mut self, name: String) -> Result<(), ServicingError> {
+    pub fn up(&mut self, name: String, force: Option<bool>) -> Result<(), ServicingError> {
+        let mut accept_prompt = Some("");
```

Comment:
I would try to avoid creating this intermediate variable that doesn't really serve much purpose. Directly use the force parameter and conditionally add the "-y" flag below.

Reply:
was not sure how to handle multiple types. like if `skip_prompt=false` I have to somehow pass that to the arg, but i can change it to make a match statement inside the arg.